dev-workflow-ui-test-data/processes/TestViewPermission.p.json

```diff
@@ -15,6 +15,9 @@
       "name" : "start.ivp",
       "config" : {
         "signature" : "start",
+        "request" : {
```

Comment:
Why this?

Reply:
ouh, that was our test before. i'll clean it